### ISSUE TITLE
Custom CSS is partly ignored on v3

### ISSUE THREAD

**Issue by jordivilar** [RocketChat/Rocket.Chat#20601]
### Description:

It seems that the custom CSS is partly ignored from a character onwards or when the code becomes automatically minified.

### Steps to reproduce:

1. Go to "admin/layout/" and click on "Custom CSS"
2. Paste custom unminified CSS in the text area
3. Click on "Save changes"

### Expected behavior:

The custom CSS is fully applied

### Actual behavior:

The custom CSS is partly applied ignoring the last classes even using `!important`

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.0
- Operating System: Debian
- Deployment Method: sources
- Number of Running Instances: 1
- NodeJS Version: 12.18.2
- MongoDB Version: 4.0.19

### Client Setup Information

- Desktop App or Browser Version: both
- Operating System: windows, linux and mac

### Additional context

Just to check if there was something wrong in the custom CSS I've tried to paste it directly in the browser element inspector overriding the minified source and it worked fine.

**Comment by gabriel-henriq** [comment]
up

**Comment by TBG-FR** [comment]
I think this is the same issue as #19405

**Comment by jordivilar** [comment]
> I think this is the same issue as #19405
> Try to add a fake CSS class at the end of the file, while we're waiting for a fix

Not really. It happens wherever you put the code not only at the end of the file. In fact, adding an unuseful class to be truncated by a bug was my idea so the guy who posted that solution and I work together.

**Comment by TBG-FR** [comment]
Could you show us the CSS classes that seem to be ignored ?

**Comment by jordivilar** [comment]
They were this two:
```
body.dark-mode .rcx-option--focus,
body.dark-mode .rcx-option:hover {
    background: rgba(255,255,255,0.1)!important;
}

body.dark-mode .rcx-option__column .rcx-status-bullet--online {
    background: #2de0a5 !important;
}
```
I tell "were" because it magically works right now O_o
